gccg_transport_api.h

```diff
@@ -81,26 +89,99 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
           "TCS": "SDR",
           "RANGE": "NARROW",
           "PAR": "12:13",
-          "alpha_included":false
-        }
+          "alphaIncluded": false,
+          "partialFrame": {
+            "width": 32,
+            "height": 32,
+            "hOffset": 132,
+            "vOffset": 132
+          },
+        },
       },
     },
     {
       "type": "audio",
-      "encodingName": "pcm",
+      "encodingName": "pcm", ## Options are: "st2110-31" or "pcm" ##
       "attributes": {
-          "channelOrder": "SMPTE2110.(SGRP)",
-          "language": "EN"
+          "totalChannels": 4      ## Total number of channels. Fixed for lifetime of connection. ##
+          "activeChannels": 4     ## Total number of active channels. Can vary, but cannot exceed totalChannels. ##
+          "channelOrder": "SMPTE2110.(SGRP)", ## Channel order string. ##
+          "language": "EN",       ## Language code. ##
+          "samplingRate": 48,     ## Sampling rate in Khz. Fixed for lifetime of connection. ##
+          "OriginalBitDepth": 24, ## Original bit depth of the samples. ##
+          "sampleCount": 100      ## Number of samples included in each channel. ##
         },
     },
     {
       "type": "ancillary-data",
-      "encodingName": "smpte291"
+      "encodingName": "rfc8331",
+      "PacketCount": 100,       ## Number of ANC packets being transported. If there is no ANC data to be transmitted
```

Comment:
Will change to "packetCount" (lowercase first character) and apply same to other JSON keys.